examples/nlp/huggingface_model/text_generation/llama/quantization/weight_only/evaluation/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Refer from https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/transformers/llm/evaluation/lm_eval
+
+from .accuracy import cli_evaluate as evaluate
```

Comment:
https://gist.github.com/justinchuby/9085242a53158f2fd7ae7aa650e55ee3 prefer not using relative imports

Reply:
Fixed. Thanks for review.